dbfit-java/core/src/main/java/dbfit/api/DBEnvironment.java

```diff
@@ -188,5 +188,6 @@ void connect(String dataSource, String username, String password,
     void connectUsingFile(String filePath) throws SQLException, IOException,
             FileNotFoundException;
 
-    DbStoredProcedureCall newStoredProcedureCall(String name, DbParameterAccessor[] accessors);
+    StatementExecution newStoredProcedureCall(String name, DbParameterAccessor[] accessors)
```

Comment:
Here we're exposing `StatementExecution` which is quite bulky in terms of public methods. Just on this API level I'm not sure the change is good.  I would rather favor a leaner interface: could be as what emerged in #541 and in longer term - maybe something like #483)